2017/JULY/JULY17/EXPTREE/EXPTREE.py

```diff
@@ -0,0 +1,57 @@
+#__BY:sirjan13__
+
+
+def gcd(n, m):
+    if n == 0:
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `python spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpp5ioqwct/2017/JULY/JULY17/EXPTREE/EXPTREE.py
+++ b/tmp/tmpp5ioqwct/2017/JULY/JULY17/EXPTREE/EXPTREE.py
@@ -25,7 +25,7 @@
     ans = (g[1] % m + m) % m
     return ans
 
- 
+
 mod1 = 10**9+7
 mod2 = mod1 + 2
 for cppp in range(int(raw_input())):
```